internal/data/receivers_wallet.go

```diff
@@ -496,3 +497,31 @@ func (rw *ReceiverWalletModel) GetByStellarAccountAndMemo(ctx context.Context, s
 
 	return &receiverWallets, nil
 }
+
+// RetryInvitationSMS sets null the invitation_sent_at of a receiver wallet.
+func (rw *ReceiverWalletModel) RetryInvitationSMS(ctx context.Context, receiverWalletId string, sqlExec db.SQLExecuter) (*ReceiverWallet, error) {
```

Comment:
The order of the method signature we try to use throughout the project starts with the dependencies and then the input params, like in `context, sqlExec, receiverWalletId`. In other words, dependencies come first and input parameters later.

```suggestion
func (rw *ReceiverWalletModel) RetryInvitationSMS(ctx context.Context, sqlExec db.SQLExecuter, receiverWalletId string) (*ReceiverWallet, error) {
```